packages/api-specification/src/window-api.html

```diff
@@ -38,13 +38,30 @@
     <button type="button" id="new-window-test" class="btn btn-default">Open Window</button>
   </form>
 
+  <hr/>
+
+  <h4>New window controls</h4>
+  <form class="form-inline">
+    <button id="close-window" type="button" class="btn btn-default">Close</button>
+    <button id="show-window" type="button" class="btn btn-default">Show</button>
+    <button id="hide-window" type="button" class="btn btn-default">Hide</button>
+    <button id="focus-window" type="button" class="btn btn-default">Focus</button>
+    <button id="blur-window" type="button" class="btn btn-default">Blur</button>
+  </form>
+
   <h3>Overview</h3>
 
   <p>The Window API is used to create a new browser window.</p>
 
   <h3>Constructor</h3>
 
   <pre>
+  new ssf.Window()
+  </pre>
+
+  <p>Creates a new window object for the current window.</p>
```

Comment:
I agree that `new ssf.Window()` is confusing. There is a static method `getCurrentWindow()` which calls this constructor, as we need the methods etc. We could have a WindowFactory class which returns new windows, or possibly subclass the window class (into `CurrentWindow` and `ExternalWindow` for example).